### ISSUE TITLE
Domains Break After Updating ubuntu/trusty64

### ISSUE THREAD

**Issue by brandenMB** [wpengine/hgv#374]
Updated my vagrant box today and every website has now gone down. I can access admin/php myadmin, and http://hgv.test/ but none of the other domains. My hosts file has the domains listed in there.

**Comment by brandenMB** [comment]
I can SSH into the box and ping the URLs just fine.

**Comment by brandenMB** [comment]
I'm seeing this message when I start:

`The guest additions on this VM do not match the installed version of VirtualBox! In most cases this is fine, but in rare cases it can prevent things such as shared folders from working properly. If you see shared folder errors, please make sure the guest additions within the virtual machine match the version of VirtualBox you have installed on your host and reload your VM.`

Which I think may be the cuase, I just don't understand what this means

**Comment by radiocat12** [comment]
I have this same issue. Has this been resolved for you?

**Comment by brandenMB** [comment]
No, in the end, I spent too much time trying to fix it so I just moved to VVV setup. This product isn't supported so I just moved on.
